<?xml version="1.0"?>
<launch>
    <!-- Test joints in UWsim with joint_state_publisher -->    
    <group ns='manual_joints'>    
    <param name="robot_description" command="$(find xacro)/xacro.py $(find freefloating_gazebo_demo)/urdf/g500arm5e.xacro"/>
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" respawn="true">
        <param name="use_gui" value="True"/>
        <remap from="joint_states" to="/g500arm5e/joint_setpoint" />
    </node>
    </group>
    
    <!-- Test body in UWsim with joint_state_publisher -->    
    <group ns='manual_body'>
    <param name="robot_description" command="$(find xacro)/xacro.py $(find freefloating_gazebo_demo)/urdf/g500_dummy.urdf"/>
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" respawn="true">
        <param name="use_gui" value="True"/>
        <remap from="joint_states" to="dummy_body_setpoint"/>
    </node>
    <node name="body_bridge" pkg="freefloating_gazebo_demo" type="body_bridge.py"/>
    </group>
    
    
</launch>
